### PR TITLE
Fixes #59 OpenAPI(Swagger) For API

### LINKED ISSUE #59: OpenAPI (Swagger) for API

### PR DESCRIPTION
**Pull request by adewolejosh**
- used a module drf_yasg to setup an OpenAPI(Swagger) to solve issue #59

### CODE REVIEW

**Comment by adewolejosh** [comment]
closes  #59

Issue fixed, all checks passed and can be reviewed to merge.

File: requirements/base.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 celery==4.4.2
+flake8==3.8.4
```
Comment (vosi) [inline requirements/base.in:2]: no need to have flake8 listed separately

File: requirements/base.in [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 flower==0.9.5
 redis==3.5.3
 tblib==1.7.0
-thenewboston==0.0.30
+thenewboston==0.0.24
```
Comment (vosi) [inline requirements/base.in:6]: please, use the latest version of the package

**Review by vosi — CHANGES_REQUESTED** [review]
also, please run scripts/compile_requirements.sh to generate requirements

File: requirements/base.in (hunk 1 above)
Comment (vosi) [reply requirements/base.in:2]: it is part of thenewboston package

**Comment by adewolejosh** [comment]
Hi @vosi I'm having some issues on my personal computer as regards the installation of the latest version of thenewboston, I don't know if it's a general issue. 

I'll be posting some pictures of the error below. The first picture is when I try to install thenewboston using pip
error

while the second picture is when I try to run script/compile_requirements.sh whilst using the latest version of thenewboston package in bash.in

error2


and I think, it is the cause for my failing tests. Anyway in which you can help?

**Comment by adewolejosh** [comment]
@vosi I believe I've made the requested changes. kindly review. Just by the way there should be a very direct message to all new contributors to use Python Versions greater than 3.7 as it is needed to run thenewboston package successfully(this was the problem hindering me from using thenewboston latest package)

**Comment by vosi** [comment]
> @vosi I believe I've made the requested changes. kindly review. Just by the way there should be a very direct message to all new contributors to use Python Versions greater than 3.7 as it is needed to run thenewboston package successfully(this was the problem hindering me from using thenewboston latest package)

https://pypi.org/project/thenewboston/ 

```
 Requires: Python >=3.7 
```

**Comment by adewolejosh** [comment]
Hi @vosi, seeing there was an upgrade in thenewboston package, I just upgraded mine to the latest. Kindly Review, Thanks!

**Comment by buckyroberts** [comment]
@danpadev can you resolve these conflicts and then review?

**Review by vosi — APPROVED** [review]

**Comment by buckyroberts** [comment]
@dmugtasimov after the [Upgrade requirements](https://github.com/thenewboston-developers/thenewboston-python/issues/48) ticket can you fix these conflicts as well?

**Pull request merged**